packages/ios_platform_images/README.md

```diff
@@ -0,0 +1,37 @@
+# IOS Platform Images
+
+A Flutter plugin to share images between Flutter and iOS.
+
+This allows Flutter to load images from Images.xcassets and iOS code to load
+Flutter images.
+
+## Usage
+
+### iOS->Flutter Example
+
+``` dart
+// Import package
+import 'package:ios_platform_images/ios_platform_images.dart';
+
+Widget build(BuildContext context) {
+  return MaterialApp(
+    home: Scaffold(
+      body: Center(
+        child: Image(image: IosPlatformImages.load("flutter")),
+      ),
+      //..
+    ),
+  );
+}
+```
+
```

Comment:
describe what this does. i.e. calls UIImage, finds image by that name in the bundle with the desired density.

Reply:
done